src/gpu/engine_info.cpp

```diff
@@ -77,7 +77,7 @@ int get_gpu_device_id()
     }
 #elif defined(__linux__)
     {
-        std::string dev_base{ "/sys/class/graphics/fb" };
+        std::string dev_base{ "/sys/class/drm/card" };
```

Comment:
May be it make sense to iterate over these two paths?

Reply:
/sys/class/graphics is more connected to frame buffer drivers, and /sys/class/drm is related to the drm module. I don't know which is more relevant to the clDNN compute. On my system, I can see all adapter details under /sys/class/drm.